su-template/default/layout/html/headers.erb

```diff
@@ -0,0 +1,15 @@
+  <meta charset="<%= charset %>">
```

Comment:
Yes, YARD HTML Helper class
[http://www.rubydoc.info/gems/yard/YARD/Templates/Helpers/HtmlHelper#charset-instance_method](http://www.rubydoc.info/gems/yard/YARD/Templates/Helpers/HtmlHelper#charset-instance_method)

This is the first line in the `"default/html/layout/headers.erb"` core template. 

Your SketchUp template had not changed it, so it was was previously used "as is" from the core default template. I needed to move a few things around (for better ordering,) and make some changes, so you should do diff against the core template file.

**This is a new `<meta>` tag for HTML5**. I wonder why it is currently in the core template as he has made it plain he doesn't wish to support HTML5 yet.